kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/enums/DeploymentType.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.systemtests.enums;
+
+public enum DeploymentType {
```

Comment:
I notice that io.skodjob.testframe.enums has InstallType.  What's the difference between this enum and that one?